applications/nrf_desktop/doc/hid_state.rst

```diff
@@ -228,10 +228,12 @@ Tracking state of transports
 
 The |hid_state| refers collectively to all transports as _subscribers_.
 
-The module tracks the state of the connected Bluetooth® LE peers and the state of USB by listening to ``ble_peer_event`` and ``usb_state_event``, respectively.
-When the connection to the host is indicated by any of these events, the |hid_state| will create a subscriber associated with the transport.
+The module tracks the state of the connected Bluetooth® LE peers and the state of USB by listening to ``hid_report_subscriber_event``.
```

Comment:
Can we use `:c::struct:` syntax also here?

Reply:
No problem